docs/update-syft.md

```diff
@@ -12,40 +12,19 @@ To do this:
     ```
 
 2. Find the [latest version of Syft][2]
-3. Pull the latest container image
+3. Install [Skopeo][3]
+4. Use [`skopeo`][4] to copy the manifest and images to our Azure Container Registry:
 
     ```
-    $ docker pull docker.io/anchore/syft:v{LATEST}
-    v0.53.4: Pulling from anchore/syft
-    0d60d5ab2113: Pull complete
-    26136f3e3dd3: Pull complete
-    497aa7f04842: Pull complete
-    Digest: sha256:37e85e8efdeaabb1b6f65c5bc175b664cb05d1aaddd0d922130b8e25d6e49726
-    Status: Downloaded newer image for anchore/syft:v{LATEST}
-    docker.io/anchore/syft:v{LATEST}
+    skopeo copy --all docker://docker.io/anchore/syft:v0.74.0 docker://governancecontainerregistry.azurecr.io/syft:v0.74.0
```

Comment:
I'd explicitly set a tag that doesn't exist so someone doesn't copy this task without checking it.

```suggestion
    skopeo copy --all docker://docker.io/anchore/syft:{LATEST} docker://governancecontainerregistry.azurecr.io/syft:{LATEST}
```